cmd/login/cmd.go

```diff
@@ -325,6 +335,10 @@ func run(cmd *cobra.Command, argv []string) {
 				clientID = args.clientID
 			}
 		}
+	} else {
+		if !args.useDeviceCode && !args.useDeviceCode {
```

Comment:
We should specifically target displaying this message only if the offline token is provided. In this case we would display the warning message for client credentials, which we don't want.

Reply:
Ack. Added offline token condition.